.gitlab-ci.yml

```diff
@@ -70,7 +70,15 @@ Test AD Integration:
     - pytest tests/
 
 
+Test OS2Sync Integration:
+  extends: .test-python
+  script:
+    - python -m unittest discover exporters/sql_export/tests 
+
+
 Test OS2Sync Integration:
   extends: .test-python
   script:
     - python -m doctest integrations/os2sync/os2mo.py
+    - python -m doctest integrations/os2sync/lcdb_os2mo.py
```

Comment:
```
python -m doctest integrations/os2sync/lcdb_os2mo.py
```
Giver:
```
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/usr/lib/python3.6/doctest.py", line 2787, in <module>
    sys.exit(_test())
  File "/usr/lib/python3.6/doctest.py", line 2775, in _test
    m = __import__(filename[:-3])
  File "integrations/os2sync/lcdb_os2mo.py", line 16, in <module>
    from integrations.os2sync import config, os2mo
  File "/home/emil/projects/mo/os2mo-data-import-and-export/integrations/os2sync/config.py", line 26, in <module>
    "OS2SYNC_XFER_CPR": top_settings["os2sync.xfer_cpr"],
KeyError: 'os2sync.xfer_cpr'
```

Reply:
Ja, der mangler jo nok en settings-fil i testmiljøet.
Det er der for det meste på udviklingsmaskinen, hvor jeg udvikler.
For at undgår at spilde tid vil jeg derfor afstå fra at lægge ting og sager i .gitlab.ci indtil det hele (incl black) kan køres på udv-maskinerne.,